### DISCUSSION TITLE
What happened to Feathub?

### DISCUSSION THREAD

**Discussion by Arcyriea** [community/community#44734] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I tried to visit this link but it doesn't seem to show the ideas page:
https://feathub.com/Akaito/reassembly-ai-mod-example/

Here is what it looks like years ago: https://web.archive.org/web/20211205122532/https://feathub.com/Akaito/reassembly-ai-mod-example/

**Comment by queenofcorgis** [comment]
Hi @Arcyriea ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
